docs/using.rst

```diff
@@ -188,6 +190,24 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant 2 way folder sync
+=========================
+For basic usage please refer `Vagrant documentation. <https://www.virtualbox.org/manual/ch04.html#sharedfolders>`_
+
+In some scenarios 2 way bi-directional sync makes more sense as it is the simplest way to move files (e.g. code) between 
+the workstation and the Vagrant guest. Here are options for bi-directional sync you can use with ADB Vagrant box.
+
+* `vagrant-sshfs <https://github.com/dustymabe/vagrant-sshfs>`_ : It works with Linux/GNU, OS X and Microsoft Windows.
+* `NFS <https://www.vagrantup.com/docs/synced-folders/nfs.html>`_ : For Linux/GNU and OS X.
+* `SMB <https://www.vagrantup.com/docs/synced-folders/smb.html>`_ : For Microsoft Windows.
+
+These solutions should work for both Virtualbox and Libvirt/KVM providers. 
+However if you are a Virtualbox user then `Virtualbox shared folder  <https://www.virtualbox.org/manual/ch04.html#sharedfolders>`_ should work fine for you. 
+
+At this point of time Virtualbox guest additions do not come pre-installed in the ADB Vagrant box. 
+Please refer to the upstream `Virtualbox documentation <https://www.virtualbox.org/manual/ch04.html>`_ to know about its installation steps. 
```

Comment:
This should really talk about vagrant-vbguest which is the easiest way to get things working, provided someone would finally merge https://github.com/projectatomic/adb-vagrant-registration/pull/70